common/common_functions.sh

```diff
@@ -1123,3 +1136,20 @@ function get_pf_device_id {
     fi
 }
 
+function label_sriov_node {
```

Comment:
nit: it's out of scope of this PR, but it seems to be good to have some generic 'label_node' function